### ISSUE TITLE
Counting Sat / Sunday

### ISSUE THREAD

**Issue by kevinprince** [geeksoflondon/hamster#99]
So looking at the new intractable stuff and can pretty much use it to do all the count @mseckington was doing previously apart from 2 which do per day counts.

Wondering if it's smart to automatically do:

arrived - for first checkin
arrived_$DAY - for each day some turns up at an event

This means we could do 

event.attended_saturday_tickets_count 

vs inspecting the key value pairs on a where query.

thoughts @cbetta @mseckington ?

**Comment by cbetta** [comment]
Yeah that's probably the way to do it right. Just create more interactions, they're cheap

**Comment by kevinprince** [comment]
Have now done this for Zebra, needs adding to Kitten. Also wrote migration to update BCB data.
